find_keywords.sh

```diff
@@ -0,0 +1,3 @@
+#/bin/bash
+
+grep -nr "b64\|base64\|cert_pem\|pem" sigstore/* test/* > key_lines.txt
```

Comment:
Nit: fine for now, but this should be dropped from the changeset once it's ready for a final review 🙂